src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null;
+};
+
+export class App extends Component<{}, State> {
+  state: State = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keyStateHandler);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keyStateHandler);
+  }
+
+  keyStateHandler = (event: KeyboardEvent) => {
+    this.setState({
+      pressedKey: `The last pressed key is [${event.key}]`,
+    });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          { !pressedKey
+            ? 'Nothing was pressed yet'
+            : pressedKey}
```

Comment:
- no need to add this space
- you should checking your booleans from the positive condition, as it is much easier to read
```suggestion
          {pressedKey
            ? pressedKey
            : 'Nothing was pressed yet'}
```